webpack/JobWizard/steps/form/SelectField.js

```diff
@@ -34,9 +34,16 @@ export const SelectField = ({
         menuAppendTo={() => document.body}
         {...props}
       >
-        {options.map((option, index) => (
-          <SelectOption key={index} value={option} />
-        ))}
+        {options[0]?.id
+          ? options.map(({ id, name }) => (
+              <SelectOption key={id} value={id}>
+                {/* Should we show id or blank? */}
+                {name || ' '}
```

Comment:
do we need both a `value` & `children` props to be passed to the `SelectOption`?

Reply:
Nope, like in the next line `<SelectOption key={index} value={option} />`.
But when an option has an id and a name the value that displays should be the name but the value that is stored should be the id.